web/pom.xml

```diff
@@ -159,7 +159,47 @@
         </dependency>
         <dependency>
             <groupId>com.amazonaws</groupId>
-            <artifactId>aws-java-sdk</artifactId>
+            <artifactId>aws-java-sdk-s3</artifactId>
```

Comment:
it makes sense to configure exclusions in `dependencyManagement` section in parent pom

Reply:
👍